arrow/api.py

```diff
@@ -21,7 +23,7 @@ def get(*args, **kwargs):
     return _factory.get(*args, **kwargs)
 
 
-get.__doc__ = _factory.get.__doc__
+update_wrapper(get, _factory.get)
```

Comment:
I was taking a look at the Python docs you linked, and it states that the `update_wrapper` function by default  "assigns to the wrapper function’s __name__, __module__ and __doc__, the documentation string) and WRAPPER_UPDATES (which updates the wrapper function’s __dict__, i.e. the instance dictionary)". Don't we just want the `__doc__` to be updated (e.g. `update_wrapper(get.__doc__, factory.get.__doc__)`)? Correct me if I am wrong though.